<launch>
  <arg name="cmd_vel" default="/cmd_vel"/>
  <arg name="name" default="turtlebot3"/>

  <param name="cmd_vel_topic_name" value="$(arg cmd_vel)"/>
  <node name="$(arg name)_follow" pkg="turtlebot3_gazebo" type="turtlebot3_follow" required="true" output="screen"/>
  <node name="create_target" pkg="turtlebot3_gazebo" type="create_target" required="true" output="screen"/>
</launch>
